### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by yurybotov**
Version 1.2.2. Fixed all important problems in 1.0.0. Tested on Dragster with 1st book code.

### CODE REVIEW

**Review by nkrkv — APPROVED** [review]
LGTM

File: src/Octoliner.h [hunk 1]
```diff
@@ -35,35 +46,41 @@ enum IOcommand {
...
-    void pwmFreq(uint16_t freq);
+    Octoliner(uint8_t i2caddress = 42);
+    void begin(uint8_t value = 208);
     void changeAddr(uint8_t newAddr);
     void changeAddrWithUID(uint8_t newAddr);
     void saveAddr();
     void reset();
-    int digitalRead(int pin);
-    int analogRead(int pin); //, uint8_t avgCount = 2);
+    int16_t analogRead(uint8_t pin); //, uint8_t avgCount = 2);
```
Comment (AlexErAmp) [inline src/Octoliner.h:55]: `//, uint8_t avgCount = 2);` - legacy comment?

File: library.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name=Octoliner
-version=1.0.0
+version=1.2.2
 author=Vasily Basalaev <vasily@amperka.ru>
 maintainer=Amperka <amperka.ru>
 sentence=A library for operation I2C 8-line-sensor module based on STM32F030F4P6 with I2Cio firmware.
```
Comment (nomad605dis) [inline library.properties]:
```suggestion
sentence=A library for operation I²C 8-line-sensor module based on STM32F030F4P6 with I2Cio firmware. 
```

**Review by nomad605dis — APPROVED** [review]
One small remark

File: src/Octoliner.h (hunk 1 above)
Comment (yurybotov) [reply src/Octoliner.h:55]: Current task is made workable version. Comments and legacy code  is future task.

Comment (AlexErAmp) [reply src/Octoliner.h:55]: Ok

**Review by AlexErAmp — APPROVED** [review]
LGTM

**Review by nomad605dis — APPROVED** [review]
LGTM

**Pull request merged**